src/style.css

```diff
@@ -0,0 +1,114 @@
+@font-face {
+  font-family: AvenirBook;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: AvenirHeavy;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+.BigSearch {
+  position: relative;
+  box-sizing: border-box;
+  height: 70px;
+  margin: 20px 0;
+}
+
+.imgBig {
+  position: absolute;
+  width: 19px;
+  left: 26px;
+  bottom: 26px;
+}
+
+.InputBigSearch::placeholder {
+  font-family: AvenirBook, sans-serif;
+}
+
+.InputBigSearch {
+  width: 100%;
+  padding: 23px 0 23px 62px;
+  font-family: AvenirBook, sans-serif;
+  font-weight: 300;
+  font-size: 16px;
+  line-height: 22px;
+  background: #fff;
+  color: #3d4e61;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.InputBigSearch:hover {
+  font-family: AvenirBook, sans-serif;
+  font-weight: 300;
+  font-size: 16px;
+  background: #fff;
+  color: #3d4e61;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.InputBigSearch:focus {
+  font-family: AvenirHeavy, sans-serif;
+  font-size: 16px;
+  background: linear-gradient(180deg,#fff 0%,#f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  outline: none;
+}
+
+.SmallSearch {
+  position: relative;
+}
+
+.imgSmall {
+  position: absolute;
+  width: 11px;
+  left: 13px;
+  bottom: 16px;
+}
+
+.InputSmallSearch::placeholder {
+  font-family: AvenirBook, sans-serif;
+}
```

Comment:
Placeholder text will automatically inherit the font family and font size of the regular input text